srcpkgs/gargoyle/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'gargoyle'
+pkgname=gargoyle
+version=2023.1
+revision=1
+build_style=cmake
+configure_args="-DWITH_TTS=on -DWITH_QT6=false"
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="SDL2-devel SDL2_mixer-devel qt5-devel fmt fontconfig-devel
```

Comment:
`fmt` looks wrong here this just install the library why would that be required?

Reply:
Honestly, it's because I didn't notice `fmt-devel` when I wrote it.

Changed it, but I'll wait until the tests are done before force-pushing it just to make sure everything's alright.